packages/SwingSet/tools/prepare-test-env-ava.js

```diff
@@ -8,6 +8,7 @@
 // https://github.com/endojs/endo/issues/768
 import '@agoric/babel-standalone';
 
+import 'ses';
```

Comment:
This may have been superfluous. I was seeing errors in assert and compartment mapper because ses didn’t initialize early enough. Root cause was the friendly linter changing the import order elsewhere.